contrib/universal-ctags/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "universal-ctags"
+pkgver = "6.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--enable-iconv",
+    "--enable-json",
+    "--enable-pcre2",
+    "--enable-xml",
+    "--enable-yaml"
+]
+make_cmd = "gmake"
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "gmake",
+    "jansson-devel",
+    "libtool",
+    "libxml2-devel",
+    "libyaml-devel",
```

Comment:
-devel ones should be moved to `makedepends=` instead

Reply:
python-docutils is used to build manpages, if I get it right, it should also be moved to `makedepends`, right?

Reply:
no; `hostmakedepends` is stuff that needs to run in the host environment, while `makedepends` are target dependencies (typically what the project links to and what will become the target package's runtime dependencies)

the distinction is important for cross-compiling, otherwise they are the same